onnxruntime/python/tools/quantization/qdq_quantizer.py

```diff
@@ -365,18 +362,9 @@ def _quantize_sharing_param_tensors(self):
                     quantized_value = self.quantized_value_map[tensor_provider_name]
                     # Quantize the input
                     initializer = find_by_name(tensor_name, self.model.initializer())
-                    if initializer is not None:
+                    if initializer:
```

Comment:
nit: explicit `is not None` is generally good

https://google.github.io/styleguide/pyguide.html#2144-decision